### ISSUE TITLE
Normalization of sensitivity measure

### ISSUE THREAD

**Issue by AlexVanMeegen** [INM-6/lif_meanfield_tools#50]
Hey!

Disclaimer: this is not an issue that leads to wrong results. It was just inconvenient for me when trying to understand the sensitivity measure (and it might lead to an insignificant performance increase).

The sensitivity measure is defined in [Bos, Diesmann, Helias 2016](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005132#pcbi.1005132.e015) as
```
Z = v^T M u / (v^T u)
```
where `u` / `v` denote the right / left eigenvector of `M` corresponding to the eigenvalue of interest.

In the [code](https://github.com/INM-6/lif_meanfield_tools/blob/080f523f25f72fbd5567295e4dadd1bbec4103fc/lif_meanfield_tools/meanfield_calcs.py#L586), the left eigenvectors are calculated using the matrix inverse of the matrix `U` containing the right eigenvectors as its columns:
```
e, U = np.linalg.eig(MH)
U_inv = np.linalg.inv(U)
...
T = np.outer(U_inv[index],U[:,index])
T /= np.dot(U_inv[index],U[:,index])
T *= MH
```
This approach assumes that `U` is invertible, i.e. the right eigenvectors are linearly independent. In this case, the inner product of left and right eigenvector is simply 1 and the normalization is irrelevant. An easy way to see this is that the left eigenvectors are the rows of `U_inv`.

Proposed fixes: Either remove the line
```
T /= np.dot(U_inv[index],U[:,index])
```
which simply divides by one or use
```
scipy.linalg.eig(MH, left=True, right=True)
```
to get the left eigenvectors without assuming that the right ones are linearly independent (the sensitivity measure does not need this assumption, see [Lancaster 1964](https://link.springer.com/article/10.1007/BF01386087)).

**Comment by AlexVanMeegen** [comment]
PS: tagging @moritzlayer and @jhnnsnk because I assume you don't come here regularly :wink:

**Comment by AlexVanMeegen** [comment]
For a prject, I did something similar to Hannah's work and there it was often the case that right and left eigenvectors were not orthogonal. Thus, I's suggest to use `scipy.linalg.eig(MH, left=True, right=True)`, what do you think? Could prepare a PR if this is of interest.

**Comment by moritzlayer** [comment]
Merged #55
